.github/ISSUE_TEMPLATE/docs-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Request
+about: General request for the Observability Documentation Team.
+title: "[Request]"
+labels: Team:Docs
+assignees: ''
+projects: 513
+
+---
+
+## Description
+*(Describe what needs to be documented. Link to relevant issues welcome)*
```

Comment:
```suggestion
*(Describe what needs to be documented. Link to relevant issues.)*
```

Would prefer to be a little more prescriptive here because the links are kind of important when we need to write the content.